util/sassRelativeImporter.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = function(url, prev, done) {
```

Comment:
_definitely_ need to document this magic - only people familiar with webpack/css-loader will have a clue what the intent is